### PR TITLE
add liquality

### PR DESCRIPTION
**Pull request by hard75**

### CODE REVIEW

File: src/components/exchange/SelectBitcoinWallet.vue [hunk 1]
```diff
@@ -105,6 +112,9 @@ export default class SelectBitcoinWallet extends Vue {
       case 'WALLET_LEDGER':
         wallet = 'ledger';
         break;
+      case 'WALLET_LIQUALITY':
+        wallet = 'liquality';
+        break;
```
Comment (ronaldsg20) [inline src/components/exchange/SelectBitcoinWallet.vue:117]: Please use the value stored in `constants` instead of the plain string

**Review by ronaldsg20 — CHANGES_REQUESTED** [review]
Please remove the `.env` file too

**Review by josedahlquist — CHANGES_REQUESTED** [review]
Please remove all the unrelated files, you may want to include them in the .gitignore file so it doesn't happen again

**Comment by jeremy-then** [comment]
> Please remove the `.env` file too

Yes, but I think it's better to revert all the commits, implement the required changes, like adding the .env to the .gitignore file, commit them and push the changes with force (-f). Because right now the .env file is in the commit history and we better remove it.

File: src/components/exchange/ConnectDevice.vue [hunk 2]
```diff
@@ -39,7 +46,7 @@
         </v-col>
       </v-row>
       <v-row class="mx-0 d-flex justify-center">
-        <v-img :src="deviceImagePath" height="300" contain />
+        <v-img v-class="{  }" :src="deviceImagePath" height="300" contain />
```
Comment (ronaldsg20) [inline src/components/exchange/ConnectDevice.vue:49]: I think It shouldn't have an empty `v-class` attribute

**Review by ronaldsg20 — APPROVED** [review]

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Pull request merged**